zipline/algorithm.py

```diff
@@ -225,6 +226,10 @@ def __init__(self, *args, **kwargs):
         # Set the dt initally to the period start by forcing it to change
         self.on_dt_changed(self.sim_params.period_start)
 
+        # The symbol lookup date specifies the date to use when resolving
+        # symbols to sids, and can be set using set_symbol_lookup_date()
+        self.symbol_lookup_date = None
```

Comment:
Could you rename this to self._symbol_lookup_date just to be clear that the field shouldn't be externally referenced?